content/concepts/dolt/sql/transaction.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Transactions
+---
+
+# Transactions
+
+## What is a Transaction?
+
+A transaction is the unit of change isolation is a database. 
+
+## How to use Transactions
+
+Transactions are generally used to manage concurrent writes to the same data in a database. If two writers attempt to write the same data, one writer is allowed to commit their transaction and the other is rolled back. Changes being made in a transaction can only be seen by the current session. 
```

Comment:
maybe note that we're READ COMMITTED? (iirc)

Reply:
We can add more later. are we READ COMMITTED?

Reply:
REPEATABLE READ, my bad